training-server.yml

```diff
@@ -1,93 +1,156 @@
-# Production tuning of server limits.conf, Postgres and NGINX required, but this will install a minimal OMERO server,
-# OMERO.web with NGINX and systemd configuration controlling both.
+# Install OMERO.server and OMERO.web with a public user on localhost
 
-# Install the latest release, including PostgreSQL on the same server
+- hosts: outreach.openmicroscopy.org
+  pre_tasks:
 
-# users-to-be-added to come from host_vars
+    - name: Install Make Movie script Prerequisite | MEncoder - Repo
+      become: yes
+      yum:
+        name:  http://li.nux.ro/download/nux/dextop/el7/x86_64/nux-dextop-release-0-5.el7.nux.noarch.rpm
+        state: present
+
+    - name: Install Make Movie script Prerequisite | MEncoder - Package
+      become: yes
+      yum:
+        name: mencoder
+        state: present
+
+    - name: OMERO.figure server-side prerequisites, script prerequisites + web server for decoupled OMERO.web
+      become: yes
+      yum:
+        name: "{{ item }}"
+        state: present
+      with_items:
+        # For OMERO.figure
+        - python-reportlab
+        - python-markdown
+        # For the 'make movie' script
+        - mencoder
+        - scipy
```

Comment:
the comment is a bit ambiguous
scipy is not for make movie but for another script (not official one yet)

Reply:
@jburel - happy to have this updated on all the playbooks where it's used. The comment was really for the line directly below it, but sure, that's not clear. Can we add comments on the same line, in YAML, @manics? i.e. append the comment after `mencoder`?

Reply:
Aha, yes, we can! https://learnxinyminutes.com/docs/yaml/

Reply:
so far we have only used that script for training
very minor comment

Reply:
@jburel  - https://github.com/openmicroscopy/prod-playbooks/pull/27